xdrip-get-entries.sh

```diff
@@ -848,6 +849,9 @@ function check_ns_calibration()
     # be entered in either UTC or local time depending on how they were entered.
     curl --compressed -m 30 -H "API-SECRET: ${API_SECRET}" "${ns_url}/api/v1/treatments.json?find\[eventType\]\[\$regex\]=Check&count=1" 2>/dev/null > $METERBG_NS_RAW
     createdAt=$(jq -r ".[0].created_at" $METERBG_NS_RAW)
+    if [ "$createdAt" == "null" ] ; then 
```

Comment:
Good catch. This does need to return. Do you use NS calibrations? I'll make sure I test them when I run this branch

Reply:
i don't "use" ns calibrations, but we do get them as a side-effect of the pump calibrations, since it eventually gets uploaded by openaps to a ns calibration... so, i see them, but more importantly i saw the error message all the time so wanted to clean that up ;)